### DISCUSSION TITLE
Members view - timezone

### DISCUSSION THREAD

**Discussion by tinola** [sinamics/ztnet#126] [General]
Hi,

When member has gone offline, there is wrong hour in tooltip.

![date](https://github.com/sinamics/ztnet/assets/3160125/64e509f8-ec51-4947-99c2-aedf382187ce)

There is 06:23, when in fact at local time it was 08:23.
Is it possible to set some environment variable to set local timezone instead of UTC?

**Comment by sinamics** [comment]
created new issue
https://github.com/sinamics/ztnet/issues/127

**Comment by sinamics** [comment]
If you have some time to spare, please consider testing the latest build on the main branch. This version should have all of your issues resolved. Your feedback would be invaluable in confirming the solution's.

Please note that this is a build from the main branch, so taking a backup beforehand is strongly recommended.

```yml
  ztnet:
    image: sinamics/ztnet:main-73f5f21
```

All changes can be seen here in the 0.3.9 Draft:
https://github.com/sinamics/ztnet/releases

**Reply by tinola** [reply]
Switched to main.
I can confirm that all your changes work flawlessly!

1. Tooltips - now show local time even without setting env TZ value, in my case CEST.
2. Sorting - the IDs and IP address columns now sort correctly.

![id_asc](https://github.com/sinamics/ztnet/assets/3160125/d90040b6-e39f-47e8-8021-d6ea333f740d)   ![ip_asc](https://github.com/sinamics/ztnet/assets/3160125/ffa20286-c9db-4764-bcab-ddda841edc03)

3. Page titles - no missing titles found.

I have some other improvements in my mind, but I'll create a new Feature Request.

I think you can upgrade `ztnet` to v0.3.9.
Thank you for your fantastic job!

**Reply by sinamics** [reply]
Awesome. Thank you for the swift feedback.

> I have some other improvements in my mind, but I'll create a new Feature Request.

Please do
